keps/2724-topology-aware-scheduling/README.md

```diff
@@ -715,6 +772,12 @@ type PodSetTopologyRequest struct {
   // For example, in the context of JobSet this value is read from jobset.sigs.k8s.io/replicatedjob-replicas.
   SubGroupCount *int32
 
+  // PodSetGroup indicates the name of the group of PodSets to which this PodSet belongs to.
+  // PodSets with the same `PodSetGroup` should be assigned the same ResourceFlavor
+  //
+  // +optional
+  PodSetGroup *string `json:"podSetGroup,omitempty"`
```

Comment:
```suggestion
  // PodSetGroupName indicates the name of the group of PodSets to which this PodSet belongs to.
  // PodSets with the same `PodSetGroupName` should be assigned the same ResourceFlavor
  //
  // +optional
  PodSetGroupName *string `json:"podSetGroupName,omitempty"`
```

It would be better to use "name" suffix since this field indicates a name of PodSerGroup instead of PodSetGroup related parameters.